src/main/java/org/fogbowcloud/manager/core/rest/controllers/ComputeOrdersController.java

```diff
@@ -1,42 +1,42 @@
-package org.fogbowcloud.manager.core.rest;
+package org.fogbowcloud.manager.core.rest.controllers;
 
 import java.util.List;
 
 import org.fogbowcloud.manager.core.models.orders.ComputeOrder;
 import org.fogbowcloud.manager.core.models.orders.Order;
-import org.fogbowcloud.manager.core.plugins.identity.exceptions.TokenCreationException;
-import org.fogbowcloud.manager.core.plugins.identity.exceptions.UnauthorizedException;
+import org.fogbowcloud.manager.core.rest.services.ComputeOrdersService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @CrossOrigin
 @RestController
 @RequestMapping(value = "compute")
 public class ComputeOrdersController {
 
+	@Autowired
+	private ComputeOrdersService computeOrdersService;
+
 	@RequestMapping(method = RequestMethod.POST)
-	public ResponseEntity<Order> createCompute(@RequestBody ComputeOrder computeOrder) {
-		return new ResponseEntity<Order>(HttpStatus.OK);
+	public ResponseEntity<Order> createCompute(@RequestBody ComputeOrder computeOrder,
+		@RequestHeader(value = "accessId") String accessId, @RequestHeader(value = "localTokenId") String localTokenId) {
+		return computeOrdersService.createCompute(computeOrder, accessId, localTokenId);
 	}
 
 	@RequestMapping(method = RequestMethod.GET)
-	public ResponseEntity<List<ComputeOrder>> getAllCompute() throws TokenCreationException, UnauthorizedException {
+	public ResponseEntity<List<ComputeOrder>> getAllCompute() {
 		return new ResponseEntity<List<ComputeOrder>>(HttpStatus.OK);
 	}
 
 	@RequestMapping(value = "/{id}", method = RequestMethod.GET)
-	public ResponseEntity<ComputeOrder> getComputeById(@PathVariable String id) {
+	public ResponseEntity<ComputeOrder> getComputeById(@PathVariable Long id) {
```

Comment:
I know that it is not part of your task, but by the fact that you are changing it the Id should be an String.